components/org.wso2.carbon.identity.branding.preference.management.core/src/main/java/org/wso2/carbon/identity/branding/preference/management/core/model/CustomText.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2023, WSO2 Inc. (http://www.wso2.com).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
Inc -> LLC

recheck the license 

```
Copyright (c) $today.year, WSO2 LLC. (http://www.wso2.com).

WSO2 LLC. licenses this file to you under the Apache License,
Version 2.0 (the "License"); you may not use this file except
in compliance with the License.
You may obtain a copy of the License at

http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing,
software distributed under the License is distributed on an
"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
KIND, either express or implied.  See the License for the
specific language governing permissions and limitations
under the License.
```

Reply:
Addressed with https://github.com/wso2-extensions/identity-branding-preference-management/pull/14/commits/ef07b96b103b15e314c5e62aac86253c030fe718